src/content/docs/sensitive-info/reference/nodejs.mdx

```diff
@@ -131,6 +133,23 @@ this value.
   </TabItem>
 </Tabs>
 
+## Accessing the body
+
+The Arcjet Sensitive Info rule accesses the request body, so if you also need to
+access the body in your application code, you should reference it before calling
```

Comment:
```suggestion
access the body in your application code, you should parse it before calling
```
I think we should say "parse" here because the dependency used is usually [body-parser](https://www.npmjs.com/package/body-parser)